src/Utilities/FlowAnalysis/FlowAnalysis/Framework/DataFlow/AbstractDataFlowAnalysisContext.cs

```diff
@@ -99,57 +98,6 @@ public abstract class AbstractDataFlowAnalysisContext<TAnalysisData, TAnalysisCo
             ValueContentAnalysisResult? valueContentAnalysisResultOpt,
             InterproceduralAnalysisData<TAnalysisData, TAnalysisContext, TAbstractAnalysisValue>? interproceduralAnalysisData);
 
-        public ControlFlowGraph? GetLocalFunctionControlFlowGraph(IMethodSymbol localFunction)
-        {
-            if (localFunction.Equals(OwningSymbol))
-            {
-                return ControlFlowGraph;
-            }
-
-            if (ControlFlowGraph.LocalFunctions.Contains(localFunction))
-            {
-                return ControlFlowGraph.GetLocalFunctionControlFlowGraph(localFunction);
-            }
-
-            if (ParentControlFlowGraphOpt != null && InterproceduralAnalysisDataOpt != null)
-            {
-                var parentAnalysisContext = InterproceduralAnalysisDataOpt.MethodsBeingAnalyzed.FirstOrDefault(context => context.ControlFlowGraph == ParentControlFlowGraphOpt);
-                return parentAnalysisContext?.GetLocalFunctionControlFlowGraph(localFunction);
-            }
-
-            // Unable to find control flow graph for local function.
-            // This can happen for cases where local function creation and invocations are in different interprocedural call trees.
-            // See unit test "DisposeObjectsBeforeLosingScopeTests.InvocationOfLocalFunctionCachedOntoField_InterproceduralAnalysis"
-            // for an example.
-            // Currently, we don't support interprocedural analysis of such local function invocations.
-            return null;
-        }
-
-        public ControlFlowGraph? GetAnonymousFunctionControlFlowGraph(IFlowAnonymousFunctionOperation lambda)
-        {
-            // TODO: https://github.com/dotnet/roslyn-analyzers/issues/1812
-            // Remove the below workaround.
-            try
-            {
-                return ControlFlowGraph.GetAnonymousFunctionControlFlowGraph(lambda);
-            }
-            catch (ArgumentOutOfRangeException)
```

Comment:
@mavasani the ticket on roslyn is marked as closed and available in v16 of VS but the unit tests are still failing with a `ArgumentOutOfRangeException` if I remove this method. Either something is still broken or I have totally failed to understand the expected fix for the ticket.